charts/alertmanager/templates/statefulset.yaml

```diff
@@ -228,6 +228,10 @@ spec:
   volumeClaimTemplates:
     - metadata:
         name: storage
+        labels:
+          {{- if .Values.persistence.labels }}
```

Comment:
why are extralabels not added here?

Reply:
Do we want this to be `extraLabels` + `persistance.labels`? That way it would be easy to add labels to everything (which is what `extraLabels` should do) and also easy to specify labels just for the PVC. 

(I don't have a use-case for that last part -- I just want a few labels on everything created by this chart. But some other charts in this repo, e.g., Prometheus, allow a lot of label customization, so I'm happy to add some customization of the PVC here.)